service/gradle/open-api.gradle

```diff
@@ -13,19 +13,16 @@ swaggerSources {
             outputDir = file(openapiOutputDir)
             components = ['models', 'apis']
             rawOptions = [
-                '--api-package',
-                "${artifactGroup}.generated.controller",
-                '--model-package',
-                "${artifactGroup}.generated.model",
-                '--model-name-prefix',
-                'Api',
-                '--additional-properties',
-                'errorOnUnknownEnum=true',
+                '--api-package', "${artifactGroup}.generated.controller",
+                '--model-package', "${artifactGroup}.generated.model",
+                '--model-name-prefix', 'Api',
+                '--additional-properties', 'errorOnUnknownEnum=true',
                 '-D',
                 'interfaceOnly=true,' +
                 'useTags=true,' +
                 'dateLibrary=java8,' +
-                'jakarta=true'
+                'jakarta=true, ' +
+                'validationMode=legacy'
```

Comment:
spring 3.3.5 adding this option and was needed to avoid introduction of new annotation 'notdefined'